src/Rasterization/Renderers/TextRenderer.php

```diff
@@ -25,7 +25,7 @@ class TextRenderer extends MultiPassRenderer
      */
     protected function prepareRenderParams(array $options, Transform $transform, ?FontRegistry $fontRegistry): ?array
     {
-        // this assumes there is no rotation or skew, but that's fine, we can't deal with that anyway
+        // only the rotational impact of skew is used
```

Comment:
I would imagine skew has no impact on the rotation? In that case, maybe reword it:

```suggestion
        // Only the rotational component of the transform is used
```

Reply:
Skew does impact rotation, for example, use -1 and 1 here in skew: https://angrytools.com/css-generator/transform/

But your wording better in both cases, so I'll use that ;)